### PR TITLE
fix sampling from MultiContinuousSpace

### PR DESCRIPTION
**Pull request by jbrea**

### CODE REVIEW

File: src/implementations/spaces/multi_continuous_space.jl [hunk 1]
```diff
@@ -21,12 +21,12 @@ Similar to [`ContinuousSpace`](@ref), but scaled to multi-dimension.
 """
 MultiContinuousSpace(low, high) = MultiContinuousSpace(promote(low, high)...)
 
-Base.eltype(::MultiContinuousSpace{T}) where {T} = T
+Base.eltype(::MultiContinuousSpace{T}) where {T} = eltype(T)
```
Comment (findmyway) [inline src/implementations/spaces/multi_continuous_space.jl]: Ah, sorry I changed the meaning of `eltype` some time ago. I prefer that `eltype` returns the type of an element in the `AbstractSpace`. Here the element of a `MultiContinuousSpace` is an `AbstractArray`. What do you think?
```suggestion
Base.eltype(::MultiContinuousSpace{T}) where {T} = T
```

File: src/implementations/spaces/multi_continuous_space.jl [hunk 2]
```diff
@@ -21,12 +21,12 @@ Similar to [`ContinuousSpace`](@ref), but scaled to multi-dimension.
...
 Base.in(xs, s::MultiContinuousSpace) =
     size(xs) == size(s.low) && all(map((l, x, h) -> l <= x <= h, s.low, xs, s.high))
 
 Base.length(s::MultiContinuousSpace) = error("MultiContinuousSpace is uncountable")
 
-function Random.rand(rng::AbstractRNG, s::MultiContinuousSpace)
-    (s.high .- s.low) .* rand(rng, size(s.low)...) .+ s.low
+function Random.rand(rng::AbstractRNG, s::MultiContinuousSpace{T}) where T
+    (s.high .- s.low) .* rand(rng, eltype(T), size(s.low)...) .+ s.low
```
Comment (findmyway) [inline src/implementations/spaces/multi_continuous_space.jl:31]: I guess with this change only https://github.com/JuliaReinforcementLearning/ReinforcementLearningEnvironments.jl/issues/55 will get fixed.

File: src/implementations/spaces/multi_continuous_space.jl (hunk 1 above)
Comment (jbrea) [reply src/implementations/spaces/multi_continuous_space.jl]: Makes sense.

File: src/implementations/spaces/multi_continuous_space.jl (hunk 2 above)
Comment (jbrea) [reply src/implementations/spaces/multi_continuous_space.jl:31]: Yes.

**Pull request merged**